opencensus/trace/tracer.py

```diff
@@ -33,34 +33,48 @@ class Tracer(object):
                     :class:`.AlwaysOnSampler`. The rest options are
                     :class:`.AlwaysOffSampler`, :class:`.FixedRateSampler`.
 
-    :type exporter: :class:`~opencensus.trace.exporters.base.exporter`
-    :param exporter: Instances of exporter objects. Default to
-                     :class:`.Printexporter`. The rest options are
-                     :class:`.Fileexporter`, :class:`.Printexporter`,
-                     :class:`.Loggingexporter`, :class:`.Zipkinexporter`,
-                     :class:`.GoogleCloudexporter`
-    """
+    :type exporters: class:`~opencensus.trace.exporters.base.exporter`
```

Comment:
This should be `type exporter` and `param exporter` still, right? Judging from the contents.

Reply:
It should! good catch.